assets/styles/app.scss

```diff
@@ -3,4 +3,44 @@
 body {
     background-color: $ground-color;
     color: $map-color;
+    margin: 0;
 }
+
+#mapid { 
+    height: 50vh; 
+    //border-radius: 45%;
+    margin-top: 50px;
+}
+
+#logo {
+    height: 200px;
+    display: flex;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.home {
+    display: grid;
+    text-align: center;
+    font-size: 25px;
```

Comment:
prefer em for font size. default font size is 16px in most cases, so 25px should be something like 1.5em